public/specifications/monitoring/MONITORING.md

```diff
@@ -0,0 +1,343 @@
+
+The associated event data dictionary supports the following keys:
+
+| Key           | Description                  | Format          | Examples  |
+|---------------|------------------------------|-----------------|-----------|
+| `browser`     | Browser information          | JSON dictionary | `{ ... }` |
+| `device`      | Device information           | JSON dictionary | `{ ... }` |
+| `media`       | Media information            | JSON dictionary | `{ ... }` |
+| `os`          | Operating system information | JSON dictionary | `{ ... }` |
+| `player`      | Player information           | JSON dictionary | `{ ... }` |
+| `screen`      | Screen information           | JSON dictionary | `{ ... }` |
+| `qoe_timings` | QoE timings                  | JSON dictionary | `{ ... }` |
+| `qos_timings` | QoS timings                  | JSON dictionary | `{ ... }` |
+
+> [!ATTENTION]
+> Requirements for each key are not provided explicitly but implementations **SHOULD** fill as much information as
+> possible. If some information cannot be reliably determined it **SHOULD** be omitted.
+
+### Browser
+
+The `browser` JSON data dictionary supports the following keys:
+
+| Field     | Description         | Format | Examples            |
+|-----------|---------------------|--------|---------------------|
+| `name`    | The browser name    | String | `Firefox`, `Safari` |
+| `version` | The browser version | String | `129.0`             |
+
+### Device
+
+The `device` JSON data dictionary supports the following keys:
+
+| Field   | Description                                  | Format                                               | Examples                               |
+|---------|----------------------------------------------|------------------------------------------------------|----------------------------------------|
+| `id`    | A unique anonymous identifier for the device | String                                               | `105124c0-fa84-4028-908e-618f2402d46f` |
+| `model` | The device model                             | String                                               | `iPhone15.7`, `Samsung Galaxy S24`     |
+| `type`  | The device type                              | `Car`, `Desktop`, `Headset`, `Phone`, `Tablet`, `TV` | `Phone`                                |
+
+### Media
+
+The `media` JSON data dictionary supports the following keys:
+
+| Field          | Description                                               | Format | Examples                                                                                 |
+|----------------|-----------------------------------------------------------|--------|------------------------------------------------------------------------------------------|
+| `asset_url`    | The URL of the content being played                       | String | `https://rts1-lsvs.akamaized.net/out/v1/62441d2399f14dce9e558b5503edba11/index.m3u8`     |
```

Comment:
I guess we should shorted URLs to something like `https://...` so that the documentation can be displayed nicely in tables.